app/src/main/java/com/fluffybros/tujefe/ui/home/HomeFragment.kt

```diff
@@ -4,31 +4,39 @@ import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import android.widget.TextView
 import androidx.fragment.app.Fragment
-import androidx.lifecycle.Observer
-import androidx.lifecycle.ViewModelProvider
-import com.fluffybros.tujefe.R
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.fluffybros.tujefe.databinding.FragmentHomeBinding
 
 class HomeFragment : Fragment() {
 
-    private lateinit var homeViewModel: HomeViewModel
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
```

Comment:
we shouldn't need this method - just pass the layout resource id into the Fragment constructor, like we did with all the other fragments.